index.html

```diff
@@ -27,9 +39,16 @@
     <script>
 
         $(document).ready(function () {
-            $("#txtbox1").ontology_search({});
-
-            $("#txtbox1").ontology_search("updateOptions", {filter: ["Cell"]});
+            $("#Stage").ontology_search({});
+            $("#Stage").ontology_search("updateOptions", {filter: ["Developmental_stage%20AND%20Homo_sapiens"]});
```

Comment:
Filters and/or issue fixed on main branch. filter: ["Developmental_stage", "Homo_sapiens"] should be working